ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__multi-nightly.yaml

```
@@ -211,6 +211,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption
+- as: aws-ipi-rootvolume-gp3-throughput-default
```

Please following the job name convention, also we can cover default* in non-destructive test, and control plane+compute in destructive test
```suggestion
- as: aws-ipi-rootvolume-gp3-throughput-f7
```